### PR TITLE
Testing scripts

### PR DESCRIPTION
**Pull request by osundwajeff**
Updated testing scripts to set up playwright in deb based distros

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 14 sec

File: playwright/staging-tests/playwright-path.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+echo "This script will discover the path to your playwright install"
+echo "If you are not in  a NixOS environment and it is not installed,"
+echo "it will try to install it."
+echo ""
+echo "At the end of calling this script , you should have a PLAYWRIGHT"
```
Comment (timlinux) [inline playwright/staging-tests/playwright-path.sh:7]: check english

File: playwright/staging-tests/playwright-path.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+echo "This script will discover the path to your playwright install"
+echo "If you are not in  a NixOS environment and it is not installed,"
+echo "it will try to install it."
```
Comment (timlinux) [inline playwright/staging-tests/playwright-path.sh:5]: what is 'it'?

File: playwright/staging-tests/playwright-path.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+echo "This script will discover the path to your playwright install"
+echo "If you are not in  a NixOS environment and it is not installed,"
```
Comment (timlinux) [inline playwright/staging-tests/playwright-path.sh:4]: double space

File: playwright/staging-tests/playwright-path.sh [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+PLAYWRIGHT="playwright"
+if [ $HAS_PLAYWRIGHT -eq 0 ]; then
+  PLAYWRIGHT="npx playwright"
+  
+  # check if it is a deb based distro
+  USES_APT=$(which apt | grep -w "apt" | wc -l)
+  NPM="npm"
+
+  if [ $USES_APT -eq 1 ]; then
+    PLAYWRIGHT_INSTALL=$($NPM ls --depth 1 playwright | grep -w "@playwright/test" | wc -l)
```
Comment (timlinux) [inline playwright/staging-tests/playwright-path.sh:31]: how do you know npm is installed?

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 9 min 0 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 36 sec

**Review by danangmassandy — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 27 sec

**Pull request merged**